src/index.html

```diff
@@ -6,9 +6,59 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+          rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
```

Comment:
Remove the 'crossorigin' attribute if not required, as it is not necessary for Google Fonts.